config/plugins.js

```diff
@@ -97,4 +97,10 @@ module.exports = ({ env }) => ({
   menus: {
     enabled: false,
   },
+  sentry: {
+    enabled: true,
+    config: {
```

Comment:
Sentry config has a `environment` var that can be used to differentiate `staging` and `production`. We already do that on other sites. Something like this: https://github.com/OriginProtocol/origin-dollar/blob/shah/heroku-deploy/dapp/src/utils/sentry.js#L18

Reply:
I just didn't see it in the plugin specific docs, but if just works the same then fair enough

Reply:
They do have it documented here: https://docs.sentry.io/clients/javascript/config/

But I agree, it's not easy to find 😅